plugin_playback_guiplayer/__init__.py

```diff
@@ -64,28 +65,46 @@ def _get_track(self, track_data):
             track = track_data[0]
             mime = track_data[1]
             mime = mime.split('/')
+            theme = track_data[2]
```

Comment:
Should we check the length of track_data before trying to access this item that potentially isn't there? 

This should mean existing Skills won't break it.

Reply:
should be fixed in latest commit

Reply:
Great, thanks!